app/models/school_data_importer.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require "file_download"
```

Comment:
Is this coming from the httpclient?

Reply:
Nah, it's a file we added, `lib/file_download`